### PR TITLE
Improve network 2.0 documentation and guidance

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: docs/technical-reference.md [hunk 1]
```diff
@@ -28,11 +28,20 @@ SSO, user identities and IAM policies, an ElasticSearch cluster, and more.
...
+  <img src="imgs/network_private.png" width="295" /> 
+</p>
+
 
 You may provide your own VPC and subnets to a Quilt stack or have the Quilt stack
-create its own subnets. In both cases Quilt uses subnets and security groups
+create its own subnets.
+
+> If you provide the VPC you are free to reuse subnets across parameters. For example:
+> `UserSubnets` and `PrivateSubnets` may refer to the same two subnets
```
Comment (akarve) [inline docs/technical-reference.md:42]:
```suggestion
> If you provide the VPC you are free to reuse subnets across parameters. For example,
> `UserSubnets` and `PrivateSubnets` may refer to the same two subnets in your VPC.
```

File: docs/technical-reference.md [hunk 2]
```diff
@@ -49,13 +70,7 @@ application load balancer
...
 OpenSearch domain
 (1/8 of the VPC CIDR)
-- (1/8 of the VPC CIDR is free)
-
-> Your Quilt instance contains _exactly one_ application load balancer that is
-> either inward or internet-facing.
-
-> If you provide the private subnets they are expected to route outbound
-> requests to AWS services via a NAT Gateway.
+- Unused
```
Comment (dimaryaz) [inline docs/technical-reference.md:73]: I'd add back the 1/8 of the CIDR here - "Unused" by itself looks confusing

**Review by dimaryaz — APPROVED** [review]

File: docs/technical-reference.md (hunk 2 above)
Comment (akarve) [inline docs/technical-reference.md:73]:
```suggestion
- Unused (1/8 of the VPC CIDR)
```

Comment (akarve) [reply docs/technical-reference.md:73]: Add fraction of CIDR for "Unused"

**Review by dimaryaz — APPROVED** [review]

**Pull request merged**